src/routes/VocabularyDetailScreen.tsx

```diff
@@ -52,26 +83,51 @@ const VocabularyDetailScreen = ({ route, navigation }: VocabularyDetailScreenPro
         closeExerciseAction={closeExerciseAction}
       />
       <DocumentImageSection document={document} />
-      <ItemContainer>
-        <WordItem answer={{ word, article }} />
-      </ItemContainer>
-      <ButtonContainer>
-        {hasNextDocument ? (
-          <Button
-            label={labels.exercises.next}
-            iconRight={ArrowRightIcon}
-            onPress={goToNextWord}
-            buttonTheme={BUTTONS_THEME.contained}
-          />
-        ) : (
-          <Button
-            label={labels.general.header.cancelExercise}
-            onPress={navigation.goBack}
-            buttonTheme={BUTTONS_THEME.contained}
-          />
-        )}
-      </ButtonContainer>
-    </Container>
+      <Container>
+        <CorrectInfoBox>
+          <WordItem answer={{ word, article }} />
+        </CorrectInfoBox>
+
+        <HorizontalLine />
+
+        <AlternativesContainer>
+          <CrystalBallIcon width={wp('8%')} height={wp('8%')} />
+          <AlternativesContent>
+            {document.alternatives.length > 0 && (
+              <>
+                <AlternativeWordHeading>{labels.exercises.vocabularyList.alternativeWords}</AlternativeWordHeading>
+                <AlternativeWords>
+                  {document.alternatives.map(value => `${value.article.value} ${value.word}`).join(', ')}
+                </AlternativeWords>
+              </>
+            )}
+
+            {/* Will be implemented in LUN-269 */}
+            <SuggestAlternativeContainer>
+              <Label>{labels.exercises.vocabularyList.suggestAlternative}</Label>
+              <ArrowRightIcon fill={theme.colors.black} width={wp('6%')} height={wp('6%')} />
+            </SuggestAlternativeContainer>
+          </AlternativesContent>
+        </AlternativesContainer>
```

Comment:
Could you please extract this or some of the stuff here to separate components please? That would make it more overseeable here and reusable.